qiskit_transpiler_service/__init__.py

```diff
@@ -10,9 +10,9 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-from .ai import AIRouting
-from .transpiler_service import TranspilerService
-from .utils import create_random_linear_function, get_metrics
+from .ai import AIRouting  # noqa: F401
+from .transpiler_service import TranspilerService  # noqa: F401
+from .utils import create_random_linear_function, get_metrics  # noqa: F401
```

Comment:
We could add this ignore `F401` to all `__init__.py` files
Here is an example: https://stackoverflow.com/questions/59167405/flake8-ignore-only-f401-rule-in-entire-file

Reply:
You're right. Fixed _a la ruff_ in https://github.com/Qiskit/qiskit-ibm-transpiler/pull/48/commits/463fee3270d6deb9ec7c0710d4c9c367418e24b1